crates/noirc_evaluator/src/brillig/mod.rs

```diff
@@ -1,16 +1,15 @@
 use std::collections::HashMap;
-
-use self::{artifact::BrilligArtifact, brillig_gen::BrilligGen};
-
-pub(crate) mod artifact;
-pub(crate) mod binary;
+#[allow(deprecated)]
 pub(crate) mod brillig_gen;
-pub(crate) mod memory;
+#[allow(deprecated)]
```

Comment:
Cannot merge with this allow deprecated lint

Reply:
These are here since I marked some methods as deprecated while refactoring, since using deprecated methods are not allowed, clippy will fail the CI

Reply:
These have now been removed